src/App.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans&display=swap');
+
+*{
+    margin: 5px;
+    padding: 0;
+}
+body{
+    background-color: #ffc74a;
+}
+header {
+    height: auto;
+    width: auto;
+    margin-bottom: 30px;
+}
+.logo{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    height:auto;
+    width:25%;
+    position: relative;
+}
+.pageButtons{
+    display: flex;
+    justify-content: center;
+}
+.pageButton1, .pageButton2{
+    background: #004E99;
+    color: white;
+    font-family: Arial;
+    font-weight: bolder;
+    padding: 10px 22px;
+    margin: 20px;
+    cursor:pointer ;
+    font-size: 100%;
+    width: 110px;
+
+}
+.App {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 50px;
+}
+button:disabled{
+    opacity: 20%
+}
+.pokemonCard {
+    padding: 10px;
+    border: 1px solid #004E99;
+    background-color: white;
+    text-align: center;
+    flex:18%;
```

Comment:
De `flex` property is een shorthand property voor: flex-grow, flex-shrink en flex-basis. Die verwacht drie waardes, zoals bijvoorbeeld `0 1 auto` waarin geen procenten worden gebruikt